requirements.txt

```diff
@@ -0,0 +1,3 @@
+bs4>=0.0.1
+selenium>=2.53.5
+Scrapy==1.1.0
```

Comment:
i like the use of scrapy here.  why invent something if someone else has already done a lot of the work.